internal/config/config.go

```diff
@@ -73,7 +79,7 @@ type UpgradeConfig struct {
 	Image string `yaml:"image"`
 	// description: |
 	//   Measurements of the updated image.
-	Measurements Measurements `yaml:"measurements"`
+	Measurements measurements.M `yaml:"measurements"`
```

Comment:
Does this need to be
```suggestion
	Measurements Measurements `yaml:"measurements"`
```
because of docgen?

Reply:
not sure why that one instance was changed.